app/models/submission.rb

```diff
@@ -9,7 +9,7 @@ class Submission < ApplicationRecord
   has_many :representations, class_name: "Submission::Representation", dependent: :destroy
   has_many :discussion_posts, class_name: "Submission::DiscussionPost", dependent: :destroy
 
-  enum tests_status: { not_queued: 0, queued: 1, passed: 2, failed: 3, errored: 4, exceptioned: 5, cancelled: 6 }, # rubocop:disable Layout/LineLength
+  enum tests_status: { not_queued: 0, queued: 1, pass: 2, fail: 3, error: 4, ops_error: 5, cancelled: 6 }, # rubocop:disable Layout/LineLength
```

Comment:
@iHiD `Submission#test_status`  and  `Submission::TestRun#status` are inconsistent. I decided to change the values here to make them consistent. Let me know what you think.

Reply:
No, these are different things. I think it should stay as it was. The submission's status have a permanence (thus the tense being "what happened"). I did think the tests should have been passed/failed/errored but I lost that debate :)

Reply:
(exceptioned is different than errored?)

Reply:
Exceptioned means that the whole test run cabundle exploded (e.g. a timeout). Errored is the "errored" state that returns from the test runner.